src/parser/monk/mistweaver/modules/features/MistweaverHealingEfficiencyTracker.js

```diff
@@ -51,12 +51,12 @@ class MistweaverHealingEfficiencyTracker extends HealingEfficiencyTracker {
   getSoothingMistDetails(spellInfo) {
     // the default tracker gets the healing of the soothing mists, but only the mana for the first cast. Every tick costs mana.
     spellInfo.manaSpent = this.soothingMist.soomTicks * SPELLS.SOOTHING_MIST.manaCost;
-    spellInfo.healingDone = spellInfo.healingDone + this.soothingMist.gustHealing;
```

Comment:
There was a bug where some modules called it 'gustHealing' and others 'gustsHealing'. I have updated so they are all 'gustsHealing' now.